website/algolia-crawler.config.mjs

```
@@ -0,0 +1,134 @@
+  rateLimit: 8,
+  maxDepth: 10,
+  maxUrls: 5000,
+  schedule: 'every 1 day at 3:00 pm',
+  startUrls: ['https://cheerio.js.org/'],
+  // Astro emits a sitemap index, not the flat /sitemap.xml Docusaurus used to.
+  sitemaps: ['https://cheerio.js.org/sitemap-index.xml'],
+  renderJavaScript: false,
+  ignoreCanonicalTo: true,
+  // GitHub Pages 301s extension-less URLs (/docs/intro -> /docs/intro/) via a
+  // plain-http hop, so http:// must be in scope or those redirects are dropped
+  // and the crawl finds nothing beyond the homepage.
```

ESLint

[error] Expected a block comment instead of consecutive line comments.

(multiline-comment-style)